hspec-core/test/Test/Hspec/Core/QuickCheck/UtilSpec.hs

```diff
@@ -117,7 +118,11 @@ spec = do
               , "Passed:"
               , "23"
               , ""
+#if MIN_VERSION_QuickCheck(2,15,0)
+              , "+++ OK, passed 2 tests (0% is 5)."
```

Comment:
Nothing has changed that should influence `parseQuickCheckResult`. It's actually just that in the recent version we decided to include labels that had not been covered in the result. Consequently, the `0% is 5` shows up here. The semantics are the same as always, if there is only one short category it will be printed in brackets with a full stop after `passed NNN tests`. So it _should_ be ok!

Reply:
Great 👍